### PR TITLE
Update condition as per JavaScript documentation.

### PR DESCRIPTION
**Pull request by Umangvaghela**
Update condition as per JavaScript documentation:

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md

### CODE REVIEW

File: client/components/post-schedule/clock.jsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ class PostScheduleClock extends Component {
 		}
 
 		if ( this.props.is12hour && amPmReference ) {
-			if ( 'undefined' === typeof modifiers.hour || ( 'PM' === amPmReference.value && modifiers.hour > 12 ) ) {
+			if ( typeof modifiers.hour === 'undefined'  || ( amPmReference.value && modifiers.hour > 12 ) === 'PM' ) {
```
Comment (alisterscott) [inline client/components/post-schedule/clock.jsx]: There is an eslint error on this line due to multiple spaces before `||`

**Comment by Umangvaghela** [comment]
hello @alisterscott 

Sir, I have added changes as per your suggestion.When you get a chance please review it.

**Comment by Umangvaghela** [comment]
@tyxla  ,@alisterscott

Can you review this PR.

Thanks

**Comment by tyxla** [comment]
Thanks @Umangvaghela!

**Pull request merged**